README.md

```diff
@@ -18,24 +18,18 @@ Milestones achieved to date and text steps can be found in the [Working Group Co
 
 The work within the DRF Community happens across a variety of different groups with a Steering Committee to oversee DRF operations, but day-to-day work happens in our Working Groups
 
-### 1️⃣ Steering Committee Structure
+### 1️⃣ Steering Committee
 <details>
   <summary>
-    <strong>🧭 Steering Committee Elections</strong>
+    <strong>🧭 Steering Committee Overview</strong>
   </summary><br />
-  Steering Committee Elections is an ongoing discussion looking for community input and is available in our [DRF forum](https://github.com/DevRel-Foundation/governance/discussions/93). As the Developer Relations Foundation community progresses in defining a Steering Committee Election Process and preparing for future elections, the current Steering Committee, composed of five members, is looking to fill two vacant seats required to meet the necessary number of representatives to meet Linux Foundation Project standards and move the foundation to fully form.
+  The Steering Committee (SC) will consist of seven (7) voting members: Five (5) Elected seats chosen by the Developer Relations Foundation (DRF) Participants; and Two (2) Appointed seats chosen by the sitting SC to maintain diversity of region, gender, and industry.
   
-The current Steering Committee will serve until new elections are held, expected by November 2025.
+In November 2025, 4 new SC members will be selected: 3 through voting and 1 through appointment. Details on the election process will be shared in September 2025.
```

Comment:
I'd recommend we keep this generic so that we don't have to update it every year. We can either link to [the charter](https://github.com/DevRel-Foundation/governance/blob/main/Technical_Charter.adoc) only (as you've done below) to provide details about the votes, or we can reference the generic details from the charter, specifically: 

- Even-numbered years: Elect two (2) elected seats and appoint one (1) seat.
- Odd-numbered years: Elect three (3) elected seats and appoint one (1) seat.